.vscode/cspell.json

```diff
@@ -178,4 +175,4 @@
     }
   ],
   "allowCompoundWords": true
-}
+}
```

Comment:
nit: add new line

Reply:
We should not require new lines for json files.